app/views/content/how-to-apply.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "How to apply for teacher training"
+heading: "How to apply"
```

Comment:
Is it worth having 'How to apply for teacher training' as the h1 as well as the title @EllieNodder, given that we know international qualified teachers are often confused about content relevant to them?